eu.cichy1173.tabela.yml

```diff
@@ -0,0 +1,29 @@
+app-id: eu.cichy1173.tabela
+runtime: org.kde.Platform
+runtime-version: '6.2'
+sdk: org.kde.Sdk
+command: tabela
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
+desktop-file-name: eu.cichy1173.eu.desktop
+modules:
+  - name: tabela
+    buildsystem: simple
+    build-commands:
+      - python -m pip install --prefix=/app .
+    sources:
+      - type: git
+        url: https://codeberg.org/cichy1173/tabela-flatpak.git
+        commit: 73753ba98060e0b8f792f306e36eafbe941b6b55
+    build-extensions:
```

Comment:
(flatpak-builder:2): flatpak-builder-WARNING **: 13:01:38.950: Unknown property build-extensions for type BuilderModule